students/ganyoa/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,16 @@
+'''
+Fizz Buzz Exercise
+-Write a program that prints the numbers from 1 to 100 inclusive.
+-But for multiples of three print “Fizz” instead of the number.
+-For the multiples of five print “Buzz” instead of the number.
+-For numbers which are multiples of both three and five print “FizzBuzz” instead.
+'''
+for i in range(1, 101):
+	a = i
```

Comment:
Looks like you're using tabs - make sure you use 4 spaces for all indentations in Python.  Most text editors have a setting automatically convert tabs to spaces.